cli/js/ops/dispatch_json.ts

```diff
@@ -1,7 +1,6 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 import * as util from "../util.ts";
 import { core } from "../core.ts";
-import { OPS_CACHE } from "../runtime.ts";
```

Comment:
Removes `OPS_CACHE` indirection which was once problematic place when trying to snapshot ES modules